docs/map.xsd

```diff
@@ -29,12 +33,78 @@
   </xs:restriction>
 </xs:simpleType>
 
+  <xs:simpleType name="percentageT">
+    <xs:restriction base="xs:double">
+      <xs:maxInclusive value="1"/>
+      <xs:minInclusive value="0"/>
+    </xs:restriction>
+  </xs:simpleType>
+
 <xs:simpleType name="nameT">
   <xs:restriction base="xs:string">
     <xs:minLength value="1"/>
   </xs:restriction>
 </xs:simpleType>
 
+  <xs:simpleType name="draworderT">
+    <xs:restriction base="xs:NMTOKEN">
+      <xs:enumeration value="index" />
+      <xs:enumeration value="topdown" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="staggeraxisT">
+    <xs:restriction base="xs:NMTOKEN">
+      <xs:enumeration value="x" />
+      <xs:enumeration value="y" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="imageformatT">
+    <xs:restriction base="xs:NMTOKEN">
+      <xs:enumeration value="png" />
+      <xs:enumeration value="gif" />
+      <xs:enumeration value="jpg" />
+      <xs:enumeration value="bmp" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="degreesT">
+    <xs:restriction base="xs:nonNegativeInteger">
+      <xs:maxInclusive value="360"/>
+      <xs:minInclusive value="0"/>
+    </xs:restriction>
+  </xs:simpleType>
+ 
+  <xs:simpleType name="opacityT">
+    <xs:restriction base="xs:double">
+      <xs:maxInclusive value="1"/>
+      <xs:minInclusive value="0"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="tileId">
+    <xs:restriction base="xs:nonNegativeInteger"/>
+  </xs:simpleType>
+
+  <xs:simpleType name="staggerindexT">
+    <xs:restriction base="xs:NMTOKEN">
+      <xs:enumeration value="even" />
+      <xs:enumeration value="odd" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="propertyT">
```

Comment:
No, all the custom simple types are simply named <name>T (T for Type). At least that's how it was when I started. So I just followed suit on that one.